hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java

```diff
@@ -406,19 +461,158 @@ void handleCommitsAnyAlias() {
         assertTrue(writableStore.modifiedAccountsInState().contains(accountID(1000L)));
         assertTrue(writableStore.modifiedAccountsInState().contains(accountID(id.accountNum())));
         assertEquals(
-                Bytes.wrap("alias"),
+                //                Bytes.wrap("alias"),
```

Comment:
Intentional?

Reply:
Forgot to remove it, thanks for catching it. Removed.